btclightclient/btc_light_client_test.go

```diff
@@ -58,6 +59,16 @@ func CommonTestCases() testCaseMap {
 	return tcs
 }
 
+func initLightClient(headers []string) *BTCLightClient {
+	decodedHeaders := make([]wire.BlockHeader, len(headers))
+	for id, str := range headers {
+		h, _ := BlockHeaderFromHex(str)
```

Comment:
error must be checked. you can pass `t *testing.T` as an argument to this function.